cli/context/store/metadata_test.go

```diff
@@ -77,16 +82,14 @@ func TestMetadataList(t *testing.T) {
 	assert.NilError(t, err)
 	defer os.RemoveAll(testDir)
 	testee := metadataStore{root: testDir, config: testCfg}
-	wholeData := map[string]ContextMetadata{
-		"simple":                    testMetadata,
-		"simple2":                   testMetadata,
-		"nested/context":            testMetadata,
```

Comment:
Is the nested feature dropped ?

Reply:
With the id derived from the name, nested contexts are actually stored flat.